etc/copyright.txt

```diff
@@ -0,0 +1,18 @@
+Copyright 2022 The Foedag team
```

Comment:
Make sure this file can be overridden in FOEDAG_rs with Rapid Silicon Copyright.

Reply:
> Make sure this file can be overridden in FOEDAG_rs with Rapid Silicon Copyright.

See PR description:
What does this pull request change?
- etc/copyright.txt has been created
  - This will be read and copied into the header
  **- Downstream clients can overwrite this with their own license info**

Reply:
I've made a PR on FOEDAG_rs that demonstrates how the copyright.txt needs to be set. I'll create one for Raptor as well once we know the copyright text we want

Reply:
Let me get in touch with Tony for advice as I don't see it in the Raptor repo.

Reply:
Waiting to hear back, until then let's create an empty one upstream as needed.